dashboard/legacy/test/middleware/test_files.rb

```diff
@@ -711,13 +712,14 @@ def test_files_copy_all
     assert_equal sound_body, last_response.body
 
     # abuse score didn't carry over
+    # note: this test doesn't actually work, since the abuse score functionality
+    # got moved to Rails.
```

Comment:
I think I see what you're saying here, but it would be nice to make it a bit more explicit. something like:
```suggestion
    # note: these assertions aren't verifying anything, since the abuse score functionality
    # got moved to Rails, so we can't actually increase the abuse score in this test.
```